app/views/revaluation.scala.html

```diff
@@ -62,8 +61,9 @@ <h1 class="govuk-heading-l">@Messages("gmp.revaluation.question")</h1>
                 InputItem(name = "revaluationDate.year", classes = s"govuk-input--width-4${if(inputHasErrors("year")){" govuk-input--error"} else {""}}", label=Some("Year"), value = revaluationForm.data.get("revaluationDate.year"))),
             fieldset = Some(Fieldset(
                 legend = Some(Legend(
+                    content = Text(messages("gmp.revaluation.question")),
                     classes = "govuk-fieldset__legend--m",
```

Comment:
the legend size should match the previous h1, so it should be `govuk-fieldset__legend--l` instead of `m`